packages/sanity/src/core/form/inputs/arrays/common/DragHandle.tsx

```diff
@@ -10,21 +10,25 @@ const DragHandleButton = styled(Button)<{grid?: boolean}>`
 
 export const SortableItemIdContext = React.createContext<string | null>(null)
 
-export const DragHandle = function DragHandle(
-  props: {
-    grid?: boolean
-  } & ButtonProps,
-) {
+interface DragHandleProps {
+  grid?: boolean
+  size: ButtonProps['size']
+  mode?: ButtonProps['mode']
+}
+
+export const DragHandle = function DragHandle(props: DragHandleProps) {
   const id = useContext(SortableItemIdContext)!
   const {listeners, attributes} = useSortable({id})
+  const {mode = 'bleed', ...rest} = props
 
   return (
     <DragHandleButton
       icon={DragHandleIcon}
-      mode="bleed"
+      tooltipProps={{content: 'Drag to reorder'}}
```

Comment:
Despite this technically being an icon-only button, adding a tooltip here can be especially distracting when you're trying to use it – suggest that we remove the tooltip here.

Reply:
What do you think about modifying the IconOnly button ts definition to accept:
`  tooltipProps: TooltipProps | 'ICON_BUTTON_NO_TOOLTIP'`
So,  in the icon only buttons that we don't want a tooltip we can use that to disable the tooltip.

```
    <DragHandleButton
      tooltipProps="ICON_BUTTON_NO_TOOLTIP" // Avoid using tooltip in this button.
      icon={DragHandleIcon}
      mode={mode}
      data-ui="DragHandleButton"
      {...attributes}
      {...rest}
      {...listeners}
    />
```

Reply:
I'm not too fond of the above approach, as i wouldn't intuitively expect `tooltipProps` to receive a string to disable it

There is value to having a tooltip for these drag handles (and in future, it's very possible that clicking these will invoke a contextual menu too). 

What might be a simpler solve is if we override `delay` in this instance and set it to something high `delay: { open: 1000 }`. WDYT?

Reply:
We could add that delay but I think we should still provide a safe opt-out from the tooltip, but making it explicit.
Maybe ` tooltipProps: TooltipProps | null ` ?

Reply:
Given that the developer should have a good reason to opt out from the tooltip in the case of icon-only buttons

Reply:
That's fair – I think `tooltipProps: TooltipProps | null` works!